File: C/src/1020.c
#include <stdio.h>

int main() {
  int days, years;
  scanf("%i", &days);
  years = days / 365;
  days %= 365;
  printf("%i ano(s)\n", years);
  printf("%i mes(es)\n", days / 30);
  printf("%i dia(s)\n", days % 30);
  return 0;
}

File: go/1020.go
package main

import "fmt"

func main() {
	var days int
	fmt.Scan(&days)
	years := days / 365
	days %= 365
	fmt.Printf("%d ano(s)\n", years)
	fmt.Printf("%d mes(es)\n", days/365)
	fmt.Printf("%d dia(s)\n", days%365)
}
